code/control/DeployPlanDispatcher.php

```diff
@@ -74,6 +74,7 @@ public function index(\SS_HTTPRequest $request) {
 	public function gitupdate(SS_HTTPRequest $request) {
 		switch($request->httpMethod()) {
 			case 'POST':
+				$this->checkSecurityToken(sprintf('%sSecurityID', get_class($this)));
```

Comment:
I believe you'll need to reset the security token from time to time to achieve Premium Security (`$this->getSecurityToken()->reset();`). The way this is implemented, if you never call `asJSON`, the token will remain the same for the entire user session.